distributed/dashboard/scheduler.py

```diff
@@ -83,6 +84,7 @@ def connect(application, http_server, scheduler, prefix=""):
     "/individual-bandwidth-types": individual_bandwidth_types_doc,
     "/individual-bandwidth-workers": individual_bandwidth_workers_doc,
     "/individual-memory-by-key": individual_memory_by_key_doc,
+    "/individual-ratio-by-key": individual_ratio_by_key_doc,
```

Comment:
Personally I don't think that the compute/transfer _ratio_ is the main point here.  I think that it's mostly compute time by key that most people will think about.  Transfer is in there just as a little extra.